### DISCUSSION TITLE
Configuration help

### DISCUSSION THREAD

**Discussion by Lodgeintel** [photoprism/photoprism#2286] [Q&A]
Hi I am running a synology server as the repository for my photos. I also run a windows 10 computer on the same network. I have a simple question can I install photo prism on my computer and have it organize the photos on the server?

**Comment by srett** [comment]
What exactly do you mean by organize? PhotoPrism can index an existing directory structure with photos and videos in it. For this you could run the PP on your Windows machine with docker-compose, and then access its web interface to browse the collection. You could also use PhotoPrism to import new photos into the collection if it has write access to the network share containing the photos. Is this roughly what you had in mind?
